test-project/Assets/Generated/Source/improbable/gdk/tests/componentswithnofields/ComponentWithNoFieldsWithCommandsTranslation.cs

```diff
@@ -463,15 +463,15 @@ public override void SendCommands(ComponentGroup commandGroup, ComponentSystemBa
                             {
                                 foreach (var request in requests)
                                 {
-                                    var schemaCommandRequest = new global::Improbable.Worker.Core.SchemaCommandRequest(ComponentId, 1);
+                                    var schemaCommandRequest = new global::Improbable.WorkerCore.SchemaCommandRequest(ComponentId, 1);
                                     global::Improbable.Gdk.Tests.ComponentsWithNoFields.Empty.Serialization.Serialize(request.Payload, schemaCommandRequest.GetObject());
 
-                                    var requestId = connection.SendCommandRequest(request.TargetEntityId,
-                                        new global::Improbable.Worker.Core.CommandRequest(schemaCommandRequest),
+                                    var requestId = connection.SendCommandRequest(request.TargetEntityId.Id,
```

Comment:
I suggest you cast this from `uint` to `long` (as it was before) to avoid any breaking changes to the user. The only reason it changed is because in 13.x, `Worker_RequestId` is `uint32_t`. In 14, we changed this to `int64_t`.

Reply:
Ah so it's going back to be a long in the future?